<?xml version="1.0"?>
<launch>
    <node pkg="ros2_ais" exec="ros_daisy_node.py" name="AIS_node" output="screen" respawn="false">
        <param name="serial_port" value="/dev/ttyUSB0" />
        <param name="frame_id" value="ais_link" />
        <param name="frequency" value="10" />
        <param name="baudrate" value="38400" />
    </node>
</launch>